apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-thumbnail-presenter.tsx

```diff
@@ -30,6 +31,7 @@ export const ResourceDetailThumbnailPresenter = ({
     <Card>
         <CardContent>
             <Grid container spacing={1} columns={{ xs: 2, sm: 4, md: 12 }}>
+                <TestIdDiv testId={id} />
```

Comment:
This is an idea for a test id component that could work across all the uses of the testid.  You add it and it inserts a div with a `data-testid` with whatever identifier you want.